omero/sysadmins/limitations.rst

```diff
@@ -43,15 +43,31 @@ user, you cannot make the group into a private group.
 File format support
 -------------------
 
+Large images
+^^^^^^^^^^^^
+
+When you import an image over a certain size, OMERO will generate a pyramid of lower resolution
+images if it doesn't already exist in the file. The threshold size is configurable using
+:property:`omero.pixeldata.max_plane_height` and
+:property:`omero.pixeldata.max_plane_width` but set to 3192x3192 pixels by
+default. However, this process can be very resource-intensive, depending on the size of the
+image as well as the image format and any data compression used, for example see
+`PixelData threads and pyramid generation issues <https://forum.image.sc/t/pixeldata-threads-and-pyramid-generation-issues/49794>`_.
+
+The OMERO pyramid generation process should be considered as deprecated and instead it is recommended
+that users avoid these issues by converting
+their data to `pyramidal OME-TIFF <https://www.openmicroscopy.org/2018/11/29/ometiffpyramid.html>`_
+files before importing into OMERO. A number of suitable tools are available such as
+`bioformats2raw & raw2ometiff<https://www.glencoesoftware.com/blog/2019/12/09/converting-whole-slide-images-to-OME-TIFF.html>`_,
+`bfconvert`, `Kheops<https://github.com/BIOP/ijp-kheops>`_, `tifffile`, `aicsimageio`, `libvips` and `QuPath`.
```

Comment:
Do we need hyperlinks for all the conversion tools/utilities?

Reply:
Well, I could find a few URLs via google, but so could anyone else. I don't know if the ones I find are more likely to be correct or stable than ones that a user might find. To be consistent, I could either add links that I find, or remove links from Kheops and link to `blog post` for bf2raw?